### DISCUSSION TITLE
如何申请加入组织？

### DISCUSSION THREAD

**Discussion by uknowzheng** [rust-lang-cn/rustwiki.org#11] [General]
看了官网的说明，很想参与组织活动，请问怎么加入

**Comment by guuzaa** [comment]
### 怎样贡献自己的翻译到《编译错误索引》
 如果我想翻译一下 [《编译错误索引》](https://rustwiki.org/zh-CN/error-index/)怎样去提交 PR？是一次性全提交还是一个索引一个索引的提交？相应的翻译文档放在哪里？

**Comment by 0xAyin** [comment]
申请加入该组织，对Rust编程饶有兴趣
